manifests/00-ingress-credentials-request.yaml

```diff
@@ -60,6 +60,8 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
 spec:
+  serviceAccountNames:
+  - ingress-operator
```

Comment:
Will this affect non-GCP providers?

Reply:
It won't as this credentials request is for GCP only. You have to make changes in the credentials request of a particular provider as done for AWS before [here](https://github.com/openshift/cluster-ingress-operator/blob/e2cdf40beea5700785cc32db32b464850bd124dd/manifests/00-ingress-credentials-request.yaml#L14)